data/changelog.yml

```
@@ -1689,3 +1689,13 @@
   Status: Released
   Entry: |
     - Bug fix: extension not reacting to logging in or out of Scratch
+
+- Version: v1.38.5
+  Index: 97
+  Tag: v1.38.5
+  Name: v1.38.5
+  Date: 2024-07-23
+  Status: Released
+  Entry: |
+    - "Show exact count" addon: temporarily removed profile exact counts for favorites, curated studios and followed studios

```

I thought this was meant to be permanent unless enough users want them to be re-added.

---

If I don't say "temporarily", then it may be understood as a final decision. If I do include it, it may be understood as a warranty of the feature coming back. What gives?

---

Maybe we can slightly break the rules this time...
```suggestion
    - "Show exact count" addon: removed profile exact counts for favorites, curated studios and followed studios (send us feedback if you were using this, we may add them back!)
```